{{cookiecutter.project_slug}}/docs/installation.rst

```diff
@@ -4,11 +4,15 @@
 Installation
 ============
 
-At the command line::
+At the command line:
+
+.. code-block:: console
 
     $ easy_install {{ cookiecutter.project_slug }}
```

Comment:
Btw, @audreyr, how do you feel about changing the default advice to recommend just `pip install {{ cookiecutter.project_slug }}` by default?